libexec/scrape

```diff
@@ -8,6 +8,8 @@ while [ $# -gt 0 ]; do
       OVERWRITE=true ;;
     -n | --dry-run )
       DRY_RUN=true ;;
+    --rbx | --rubinius )
+      RUBIES+=('rubinius') ;;
```

Comment:
What is the minimum supported version of bash for rbenv? I see a lot of old school array-pushing in other pieces of the codebase. e.g. `before_hooks["${#before_hooks[@]}"]="$hook"`

If the lowest supported version is 3.2 or higher, we could improve the clarity of the array-pushing.